llvm/include/llvm/Transforms/Utils/SampleProfileInference.h

```diff
@@ -28,6 +28,7 @@ struct FlowBlock {
   uint64_t Weight{0};
   bool HasUnknownWeight{true};
   bool IsUnlikely{false};
+  bool IsExit{false};
```

Comment:
This is going to be confusing as we already have `isExit` based on whether there is outgoing edges. What's different for the boolean, and what's our definition of "is exit block"?

Reply:
I agree, just changed this representation